README.md

```diff
@@ -106,7 +106,28 @@ If you want to remove a volume (e.g. to start with a fresh database or redis cac
 
 ### Resetting Docker:
 
-In some cases Docker container image versions may get stale and may need to be rebuilt with no cache. To completely remove all docker images, run the system prune command.
+In some cases Docker container image versions may get stale and may need to be rebuilt with no cache. Attempt the steps in option 1 first before proceeding to option 2.
+
+#### Option 1: Rebuild images
+
+Step 1: Stop and remove all running docker containers for POD
+```
+docker compose stop
+```
```

Comment:
this stops but doesn't remove, right? might want to also add `docker compose rm`

Reply:
Good catch, I've updated the command to docker compose down which stops and removes the containers.